std/stdio.d

```diff
@@ -3258,14 +3258,20 @@ struct LockingTextReader
 }
 
 /**
- * Indicates whether $(D T) is a file handle of some kind.
+ * Indicates whether $(D T) is a file handle, i.e. the type
+ * is implicitly convertable to $(LREF File) or a pointer to a
+ * $(REF FILE, core,stdc,stdio).
+ *
+ * Returns:
+ *      `bool`
```

Comment:
I think an obvious boolean doesn't need a `Returns` section. And this isn't a function, so `Returns` isn't exactly accurate.

I think if you just change it to "A `bool` indicating whether...", it should be fine.